### PR TITLE
MES-2920: Performance Issue Search Results

### PR DESCRIPTION
**Pull request by szabolcs-keresztes**
Performance issue when loading more than 30 test results has been solved.

### CODE REVIEW

File: src/pages/test-results-search/test-results-search.scss [hunk 1]
```diff
@@ -66,4 +66,36 @@ page-test-results-search {
...
+    position: absolute;
+    width: 800px;
+    z-index: 100;
+  }
+
+  @media screen and (min-width: 900px) {
+    .search-inputs {
+      position: absolute;
+      width: 1080px;
+      z-index: 100;
```
Comment (ammarhaiderbjss) [inline src/pages/test-results-search/test-results-search.scss]: Did we need still this z-index then?

File: src/pages/test-results-search/test-results-search.ts [hunk 2]
```diff
@@ -101,4 +102,11 @@ export class TestResultsSearchPage extends BasePageComponent {
       .subscribe();
   }
 
+  myHeaderFn(record: any, recrodIndex: any): string {
```
Comment (ammarhaiderbjss) [inline src/pages/test-results-search/test-results-search.ts]: typo here

File: src/pages/test-results-search/test-results-search.scss (hunk 1 above)
Comment (ammarhaiderbjss) [reply src/pages/test-results-search/test-results-search.scss]: ignore me, thought this was .list-header

File: src/pages/test-results-search/test-results-search.ts (hunk 2 above)
Comment (ammarhaiderbjss) [reply src/pages/test-results-search/test-results-search.ts]: What's this method for btw?

File: src/pages/test-results-search/test-results-search.scss (hunk 1 above)
Comment (szabolcs-keresztes) [reply src/pages/test-results-search/test-results-search.scss]: The z-index would bring the search tabs in from of the the ion-list header.

File: src/pages/test-results-search/test-results-search.ts (hunk 2 above)
Comment (szabolcs-keresztes) [reply src/pages/test-results-search/test-results-search.ts]: The `ion-list` needs some function to determine which items in the list need header. In our case just the first item will need header, but we purely use this only for design so we don't need to return any information just an empty string.

Corrected the typo.

**Pull request merged**